README.md

```diff
@@ -32,7 +32,7 @@ $ bundle
 In `config/application.rb`, add:
 
 ```bash
-require "action_view/component"
+require "action_view/component/railtie"
```

Comment:
Note: we should do a minor version bump with this change.

Reply:
Let's keep the require `railtie` naming convetion on the README file. To keep backward compatibility, the older require `railtie` statement was included on `component.rb` file (see the last commit).